somnio_starter/__brick__/{{project_name.snakeCase()}}/packages/user_repository/test/src/models/user_test.dart

```diff
@@ -30,5 +33,22 @@ void main() {
       expect(userA, equals(secondUserA));
       expect(userA, isNot(equals(userB)));
     });
+
+    test('fromJson and toJson', () {
+      final json = <String, dynamic>{
+        'id': validId,
+        'email': validEmail,
```

Comment:
same comment here

Reply:
Does it applies here? We're setting a json with the variables